libraries/ESP8266WebServer/src/ESP8266WebServer-impl.h

```diff
@@ -831,4 +846,37 @@ String ESP8266WebServerTemplate<ServerType>::responseCodeToString(const int code
     return String(r);
 }
 
+template<typename ServerType>
+void ESP8266WebServerETagTemplate<ServerType>::serveStaticETag(const char* uri, FS& fs, const char* path, const char* cache_header){
+  WST::_addRequestHandler(new StaticRequestETagHandler<ServerType>(fs, path, uri, cache_header));
+}
+
+template<typename ServerType>
+void ESP8266WebServerETagTemplate<ServerType>::serveStaticETag(const char* uri, FS& fs, const char * path) {
+
+  File toHash = fs.open(path, "r");
+
+  if(toHash){
+
+    MD5Builder calcMD5;
+    calcMD5.begin();
+
```

Comment:
It's minor and a taste thing, but unless you're being paid by the KLOC [old IBMer? :) ] what's the reason for the skipping every other line?  Could it be a linefeed issue in an editor?